### DISCUSSION TITLE
Control frame rate of video recording?

### DISCUSSION THREAD

**Discussion by ludazhao** [cypress-io/cypress#21523] [Questions and Help]
Hi Cypress team!

Is there a way to specify the frame rate of the video recording for different tests? We have a test that takes quite significant time to run (~5 minutes), vs. tests that are quite fast (<10 seconds). They currently are all using the same frame rate, which means that the shorter tests have very short clips consisting of only a few frames, making them not really useful for viewing. Is there a way to control the framerate of the videos per test? Thanks so much!

**Comment by gplusdotgr** [comment]
https://docs.cypress.io/guides/references/configuration#Videos confident enough there's no such setting, but I do have a workaround; 
(maybe not recommended as a best practice but will help your case), add some arbitrary waits? < 10 second tests sounds a bit 'too' fast, I wonder what they do, maybe combining them with others would make more sense? 
Sorry for not giving you a solid solution;

**Comment by yogithesymbian** [comment]
Any update or news ?
